test/unit/setup.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const PACKAGE_ROOT = '../../'
+const FileStorageRequire = file => require(path.join(PACKAGE_ROOT, file))
+const http = require('http')
+
+// setup authentication endpoint
+function authServer (config = {}) {
+    const host = config.host || 'localhost'
+    const port = config.port || 3002
+    const authConfig = config.authConfig || [
+        { token: 'test-token', projectId: 'test-project' }
+    ]
+    const requestListener = function (req, res) {
+        try {
+            let authToken
+            const urlParts = req.url.split('/')
+            const projectId = urlParts.pop()
+            const route = urlParts.join('/')
+            switch (route) {
+            case '/account/check/project':
+                authToken = authConfig.find(auth => auth.projectId === projectId)
+                if (req.headers.authorization === ('Bearer ' + authToken.token)) {
+                    res.writeHead(200)
+                    res.end('{}')
+                    return
+                }
+                throw new Error('Unknown request')
+            default:
+                res.writeHead(404)
+                res.end(JSON.stringify({ error: 'Resource not found' }))
+            }
+        } catch (error) {
+            res.writeHead(401)
+            res.end(JSON.stringify({ error: 'unauthorised' }))
+        }
+    }
+
+    const authServer = http.createServer(requestListener)
+    authServer.listen(port, host, () => {
+        // listening for requests on port 3002
+    })
+    return authServer
+}
+
+async function setupApp (config = {}) {
+    process.env.FF_FS_TEST_CONFIG = `
```

Comment:
Where is this used?

Reply:
`FF_FS_TEST_CONFIG` is used for passing test config to the flowforge-file-server app here: https://github.com/flowforge/flowforge-file-server/blob/b22fc4bf7b394a3ccda9f893f381aaa51cff4c08/forge/config.js#L8

In case it isnt clear...
* the tests setup a fake FF auth endpoint at localhost:3002 an responds to `/account/check/project/project-id` requests (rather than fire up a forge app)
  * NOTE: this fakery still expects a paired `projectId` and `token` as passed in to its `authConfig` option in `auth_spec.js` at line:8 
* this allows us (in [test/unit/auth_spec.js](https://github.com/flowforge/flowforge-file-server/pull/20/files/2366538d7d24984db53a1ed18673e55fa001d46f#diff-7a0d8721928ff56bee303a2b4ed84bdddd6165a801546726709bd57431e38d92)) to call the the endpoints (for example `/v1/files/test-team-1/test-project-1/test.txt`) 
* NOTE: by faking the auth endpoint, we are still calling and testing `checkAuth()` in this app in `forge/auth.js`